### DISCUSSION TITLE
Unhandled exception Failed to retrieve AI model

### DISCUSSION THREAD

**Discussion by StagSaga** [silveredgold/beta-protection#85] [General]
I cant get this to run. I keep getting this message. 

Unhandled exception. System.IO.InvalidDataException: Failed to retrieve AI model! Aborting...
   at Program.<Main>$(String[] args) in /home/runner/work/beta-censoring/beta-censoring/src/BetaCensor.Server/Program.cs:line 40
   at Program.<Main>(String[] args)

**Comment by nooblobby1337** [comment]
same

**Comment by jimojo** [comment]
I believe the solution you need is posted here;
[[https://github.com/silveredgold/beta-censoring/issues/39](https://github.com/silveredgold/beta-censoring/issues/39#issuecomment-2174694370)](url)

This worked for me.

(idk why, but attaching the link leads to a page error even though the page is still available. Copy-pasting seems to work so try that instead, or manually navigate through the creator's page ( silveredgold ) to the beta censoring closed issues page.

https://github.com/silveredgold/beta-censoring/issues/39#issuecomment-2174694370

**Reply by nooblobby1337** [reply]
thank you.
